TFT/src/User/Menu/common.c

```diff
@@ -199,55 +199,6 @@ void drawStandardValue(const GUI_RECT *rect, VALUE_TYPE valType, const void *val
   GUI_SetBkColor(origBgColor);
 }
 
-bool warmupTemperature(uint8_t toolIndex, void (* callback)(void))
```

Comment:
I put those functions here (menu directory) because they also use UI (showDialog etc...). In the API directory no reference to GUI should be present

Reply:
Fixed